pkg/utils/protocols/protocols_test.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2025 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package protocols
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetProtocolNameByNumber(t *testing.T) {
+	protocols := []struct {
+		description string
+		number      int
+		name        string
+		ok          bool
+	}{
+		{
```

Comment:
Would add another test with `-1` and maybe `TCP` since `TCP` will be the most used one for Inspektor Gadget